pkg/cli/debug.go

```diff
@@ -1631,10 +1634,37 @@ func initPebbleCmds(cmd *cobra.Command, pebbleTool *tool.T) {
 			pebbleCryptoInitializer(cmd.Context())
 			return nil
 		}
+		if c.Name() == "excise" {
+			f := c.Flags()
+			f.Uint64Var(&debugPebbleOpts.exciseTenantID, "tenant-id", 0, "tenant ID for table to excise (must be used in conjunction with --table-id)")
+			f.Uint32Var(&debugPebbleOpts.exciseTableID, "table-id", 0, "table ID to excise (must be used in conjunction with --tenant-id)")
+		}
 		initPebbleCmds(c, pebbleTool)
 	}
 }
 
+// pebbleExciseSpanFn implements tool.DBExciseSpanFn. It returns a span
+// for a table if the table/tenant ID flags are used.
+func pebbleExciseSpanFn() (pebble.KeyRange, error) {
+	if debugPebbleOpts.exciseTenantID == 0 && debugPebbleOpts.exciseTableID == 0 {
+		return pebble.KeyRange{}, nil
+	}
+	if debugPebbleOpts.exciseTenantID == 0 || debugPebbleOpts.exciseTableID == 0 {
```

Comment:
There might also be value in blowing up the whole tenant, right? in which case we should be able to specify just the tenant ID and not the table ID?

Reply:
CC @msbutler WDYT?

Reply:
yeah, this could be nice in a UA cluster, if we needed to wipe the main tenant.

Reply:
Done. I added an `--entire-tenant` flag which must be used in conjunction with `--tenant-id` (to make accidents less likely). We disallow wiping the entire system tenant.